week-02-ajax-and-js-objects/day-02/drill/readme.md

```diff
@@ -0,0 +1,48 @@
+## HTML Strings
+
+For the following challanges you are going to use the template from https://github.com/sf-wdi-24/frontend-template
+
+1. From the template repo copy the url to clone it
+2. In your terminal, go into your dev folder and type 'git clone' + paste the link
```

Comment:
Can we suggest `git clone` + link + `html-strings` so they can see how to change a repo name when cloning?